pkg/virt-controller/watch/migration/migration_test.go

```diff
@@ -360,6 +362,13 @@ var _ = Describe("Migration watcher", func() {
 		return migrationConfiguration
 	}
 
+	sanityExecute := func() {
+		watchtesting.SanityExecute(controller, []cache.Store{
+			controller.vmiStore, controller.podIndexer, controller.migrationIndexer, controller.nodeStore,
+			controller.pvcStore, controller.pdbIndexer, controller.migrationPolicyStore, controller.resourceQuotaIndexer,
+		}, Default)
```

Comment:
Recently, `storageClass` and `storageProfile` caches were introduced. Can you cover those too?
This is a perfect example of how easy we can step back from covering all the cache corruption

Reply:
Sure